src/main/java/nl/tudelft/ti2206/group9/gui/scene/SettingsScene.java

```diff
@@ -23,37 +23,53 @@
     enum BType {
         /** Back button. */
         SETTINGS_BACK,
-        /** Sound toggle. */
-        SETTING_SOUND
+        /** Sound Effects toggle. */
+        SETTING_SOUNDEFFECTS,
+        /** Soundtrack toggle. */
+        SETTING_SOUNDTRACK
      }
 
     /** Toggle width. */
-    private static final int TOGGLE_WIDTH = 150;
+    private static final int TOGGLE_WIDTH = 270;
     /** Toggle text size. */
     private static final int TOGGLE_TEXT_SIZE = 16;
+    /** Initially the toggles are set to off. */
+    private static final String INIT_TOGGLE = "OFF";
+
     /**
      * Creates a Sound toggle button and a Back button.
      * @return an array of Nodes to be added to the Scene.
      */
     @Override
     public Node[] createContent() {
-        final Button backButton = createButton("BACK", 0, 25);
+        final Button backButton = createButton("BACK", 5, 24);
 
-        String soundToggle = "OFF";
-        if (State.isSoundEnabled()) {
-            soundToggle = "ON";
+        String soundtrToggle = INIT_TOGGLE;
+        if (State.isSoundtrackEnabled()) {
+            soundtrToggle = "ON";
+        }
+        final Button soundtrButton = createButton("Soundtrack: "
+        + soundtrToggle, 5, 14);
+        soundtrButton.setFont(Style.getFont(TOGGLE_TEXT_SIZE));
+        soundtrButton.setPrefWidth(TOGGLE_WIDTH);
+        String soundEfToggle = INIT_TOGGLE;
+        if (State.isSoundEffectsEnabled()) {
+            soundEfToggle = "ON";
         }
-        final Button soundButton = createButton("Sound: " + soundToggle, 2, 18);
-        soundButton.setFont(Style.getFont(TOGGLE_TEXT_SIZE));
-        soundButton.setPrefWidth(TOGGLE_WIDTH);
+        final Button soundEfButton = createButton("Sound effects: "
+        + soundEfToggle, 5, 17);
```

Comment:
This file looks like it can use a Ctrl-A Ctrl-I as well. Lines 52 and 60 have incorrect indentation.

<review, layout